### PR TITLE
Add requestroom aliases

### PR DESCRIPTION
**Pull request by TheRun98**

### CODE REVIEW

File: spacialgaze-plugins/roomrequest.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ let allowRequests = true;
 
 exports.commands = {
 	confirmrequestroom: 'requestroom',
+	roomrequest: 'requestroom',
```
Comment (ghost) [inline spacialgaze-plugins/roomrequest.js]: I feel like this shouldn't be an Alias. :/

Comment (DeathlyPlays) [reply spacialgaze-plugins/roomrequest.js]: Yeah, it's too close to another command

Comment (TheRun98) [reply spacialgaze-plugins/roomrequest.js]: What command? :/

Comment (ghost) [reply spacialgaze-plugins/roomrequest.js]: roomrequests

**Pull request merged**